inject.js

```diff
@@ -371,7 +375,12 @@
       });
       observer.observe(document, { childList: true, subtree: true });
 
-      var videoTags = document.getElementsByTagName('video');
+      if (tc.settings.audioBoolean) {
+        var videoTags = document.querySelectorAll('video,audio');
```

Comment:
nitpicking, but since we're now picking up both audio and video, can we rename the var to `mediaTags`?

Reply:
It bugs me too. Do you want to change it everywhere? Lots of references to "video" in videospeedcontroller lol.